prdoc/pr_3867.prdoc

```diff
@@ -0,0 +1,9 @@
+title: Remove unused NonConcrete error in asset transfer
```

Comment:
I'd say this PR doesn't need a PRDoc, it's just cleanup. If you add the R0-silent label then CI won't check for a PRDoc. I can add it if you agree.

Reply:
Thank you for your feedback. I removed the prdoc in this commit: https://github.com/paritytech/polkadot-sdk/pull/3867/commits/d1c65ce6ca25b919eb491bb32c64f442d6260cec

As I don't have a permission, could you help me to label this PR as `R0-silent`. Thanks